### ISSUE TITLE
How do I set dynamically different font size for each item in series of Treemap chart of amchart4

### ISSUE THREAD

**Issue by VatsalBhavsar** [amcharts/amcharts4#1170]
I am using amchart4 charting library to create a Treemap chart. I've created bullets to show some additional text on each item of series. How do I set different font size for each item's label in series according to the size of the item.

**Comment by bosiljkakostic** [comment]
in the place where you create the size of bullet define also bullet.fontSize, or add adapter visible, set bullets fontSize, and return true (for visible)

**Comment by VatsalBhavsar** [comment]
```
    var bullet1 = level1.bullets.push(new am4charts.LabelBullet());
    bullet1.label.text = "{valLabel}";
    bullet1.locationX = 1;
    bullet1.locationY = 1;
    bullet1.dx = 40;
    bullet1.dy = 20;
    bullet1.horizontalCenter = 'left'
    bullet1.label.fontSize = "{fontSize}";
    bullet1.fontWeight = 'bold'
    bullet1.label.fill = am4core.color("#fff");
```
While creating data for chart I've added "fontSize" property for each object and fontSize for every object is different.

When I try to access font size as shown in the above code, it doesn't accept it that way. I also tried to give random numbers using Match.random() as shown below,

`    bullet1.label.fontSize = Math.floor(Math.random() * 10);`

 to check if "bullet.label.fontSize" accepts it or not. I found that the first random number generated is applied as font size for all the label.

Just to be clear I'll explain once again what exactly I need.

![treemapChart](https://user-images.githubusercontent.com/25295796/55721661-7f705680-5a21-11e9-884b-d53e55b195c5.png)

Above attached is the image of a treemap chart which has label in it. I need to set the font size of those labels according to the size of its container section. The bigger the section is, bigger the font size of it's label should be.

Thanks for the help.
